### DISCUSSION TITLE
Create a new file

### DISCUSSION THREAD

**Discussion by skytylon** [iterate-ch/cyberduck#14836] [Q&A] [answered]
When the tree structure is deployed, it is impossible to create a new file with the same name in a sub-folder.

![image](https://github.com/iterate-ch/cyberduck/assets/38259402/d30a7f94-a722-408d-bbe1-0dd5e4dec017)

**Comment by dkocher** [comment, marked as answer]
This is expected behaviour and matches what happens in Windows Explorer. You have to navigate into the folder prior creating a new file. The file is created in the current working directory regardless of the selection.
